libunwind/src/UnwindRustSgxSnprintf.c

```diff
@@ -10,7 +10,9 @@
 #pragma GCC diagnostic ignored "-Wfloat-conversion"
 #pragma GCC diagnostic ignored "-Wsign-conversion"
 #pragma GCC diagnostic ignored "-Wstrict-overflow"
+#if __GNUC__ >= 7
```

Comment:
Why do we need this?

Reply:
So that it builds both on my laptop and in Rust CI